portfolio/src/main/webapp/script.js

```diff
@@ -38,13 +38,13 @@ function randomNumber() {
  */
 function secretPage(){
   //Generates a number between 1 and 200 for both numbers.
-  var a = randomNumber();
-  var b = randomNumber();
+  let a = randomNumber(200,1);
+  let b = randomNumber(200,1);
 
 
   //If b is a greater number we flip the values
-  if(b > a){
-    var hold = b;
+  if (b > a) {
+    let hold = b;
```

Comment:
`let` can be `const` here since the variable `hold` is not reassigned after initialization.